### ISSUE TITLE
Is anyone able to do full-parameter fine-tuning with Hunyuan Video

### ISSUE THREAD

**Issue by changbolu22** [a-r-r-o-w/finetrainers#266]
lora works fine. but when it comes full parameter finetuning, the model crashes after tens of steps.
I guess it might because the original training loss in the paper contains the perceptual loss and the gan loss (i might be wrong)?
anyone have a clue ?

**Comment by a-r-r-o-w** [comment]
By model crasing, do you mean it's predictions become bad or the training fails? I have yet to account for the other losses mentioned in the paper -- it could definitely be a reason for collapse. Will update soon

**Comment by KurokyLee** [comment]
> lora works fine. but when it comes full parameter finetuning, the model crashes after tens of steps. I guess it might because the original training loss in the paper contains the perceptual loss and the gan loss (i might be wrong)? anyone have a clue ?

could you please tell me the vram use when you trained hunyuan video with full parameter?

**Comment by KurokyLee** [comment]
> lora works fine. but when it comes full parameter finetuning, the model crashes after tens of steps. I guess it might because the original training loss in the paper contains the perceptual loss and the gan loss (i might be wrong)? anyone have a clue ?

@changbolu22  could you please tell me the vram use when you trained hunyuan video with full parameter?

**Comment by a-r-r-o-w** [comment]
So I read the paper again and there are no additional losses to account for in the current trainer. The losses that you mentioned are part of the training objective of the VAE. Here, we keep the VAE frozen and only train the transformer, which requires minimizing the MSE between true velocity and model predicted velocity (standard flow matching prediction task).

**Comment by viiika** [comment]
@changbolu22 Did you solve this model crash issue when finetune hunyuanvideo with full parameters?
